Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: migrate
+
+migrate:
+	cp out/artifacts/aio/aio.jar ~/OSBot/Scripts/
```

Comment:
Can you remove this file?

Reply:
Sure I'll add it to the .gitignore, I use it to speed up local development.